espflash/src/flasher/mod.rs

```diff
@@ -1102,9 +1099,9 @@ impl Flasher {
             });
         }
 
-        let mut target = self
-            .chip
-            .flash_target(self.spi_params, self.use_stub, false, false);
+        let mut target =
+            self.chip
+                .flash_target(self.spi_params, self.use_stub, self.verify, self.skip);
```

Comment:
I tried to test this (`save-image_write-bin.sh`) and I can see both `self.verify` and `self.skip` are `true` now - just double checking if it was intended change or not

Reply:
It looks like it hasn't changed since the original impl in https://github.com/esp-rs/espflash/pull/538, cc: @bjoernQ, any reason why this was done this way?

Reply:
t.b.h. I don't remember why `write_bin` didn't have a skip or a verify command line flag